tests/integration/targets/docker_network/tasks/tests/overlay.yml

```diff
@@ -60,3 +60,55 @@
     docker_swarm:
       state: absent
       force: true
+
+####################################################################
+## ingress #########################################################
+####################################################################
+
+- block:
+  # Overlay networks require swarm initialization before they'll work
+  - name: swarm
+    docker_swarm:
+      state: present
+      advertise_addr: "{{ ansible_default_ipv4.address | default('127.0.0.1') }}"
```

Comment:
Please do not put the Daemon into Swarm mode multiple times in the same test. This caused a lot of problems in the past. Instead rewrite this file so that Swarm is enabled exactly once and disabled exactly once.